pkg/destroy/alibabacloud/alibabacloud.go

```diff
@@ -834,19 +863,20 @@ func (o *ClusterUninstaller) deleteSecurityGroups() (err error) {
 		},
 	)
 
+	logger.Info("Security groups deleted")
 	return
 }
 
-func (o *ClusterUninstaller) deleteSecurityGroup(securityGroupID string) (err error) {
-	o.Logger.Debugf("Start to delete security group %q", securityGroupID)
+func (o *ClusterUninstaller) deleteSecurityGroup(securityGroupID string, logger *logrus.Entry) (err error) {
+	logger.WithField("id", securityGroupID).Debug("Deleting")
 	request := ecs.CreateDeleteSecurityGroupRequest()
 	request.SecurityGroupId = securityGroupID
 	_, err = o.ecsClient.DeleteSecurityGroup(request)
 	return
 }
 
-func (o *ClusterUninstaller) deleteSecurityGroupRules(securityGroupID string) (err error) {
-	o.Logger.Debugf("Start to delete security group %q rules ", securityGroupID)
+func (o *ClusterUninstaller) deleteSecurityGroupRules(securityGroupID string, logger *logrus.Entry) (err error) {
+	logger.WithField("securityGroupID", securityGroupID).Debug("Revoking")
```

Comment:
Other places where the security group ID is added as a field, the key "id" is used. Let's stay consistent. Either change everywhere else to use "securityGroupID" or change this place to use "id".
```suggestion
	logger.WithField("id", securityGroupID).Debug("Revoking")
```

Reply:
Have updated^^